src/components/Catalog/catalog.jsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from "react";
+import database from "./Images/database-icon.png";
+import smartphone from "./Images/smartphone-icon.png";
+import code from "./Images/code-icon.png";
+import layout from "./Images/layout-icon.png";
```

Comment:
- [ ] The path to these files are incorrect. There was an error message when I tested your app cos of their path. It should be like this. :point_down: 

```
  import database from "../../Images/database-icon.png";
  import smartphone from "../../Images/smartphone-icon.png";
  import code from "../../Images/code-icon.png";
  import layout from "../../Images/layout-icon.png";

```